frontend/src/css/BookingView.css

```
@@ -1316,3 +1227,29 @@
 .fc .fc-toolbar-title {
   font-size: 0.875rem !important;
 }
+
+/* Suggestion events - read-only appearance (no hover effect) */
+:deep(.fc-event.suggestion-exact),
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)